grc/core/params/param.py

```diff
@@ -176,6 +190,59 @@ def is_float(self, num):
         except ValueError:
             return False
 
+
+    def pretty_print(self):
+        """
+        Get the repr (nice string format) for this param.
+
+        Returns:
+         the string representation
+        """
+
+        from ...gui import Utils
+        ##################################################
+        # Simple conditions
+        ##################################################
+        value = self.get_value()
+        if not self.is_valid():
+            return self.truncate(value)
```

Comment:
Isn't truncate a member of the `gui` param, but not of the `core` param`?

Reply:
Good spot! I moved that over to `core/param` now :+1: